Makefile.in

```diff
@@ -93,73 +93,69 @@ distclean: clean
-	install -c -m 0755 contrib/jail_prober.py $(DESTDIR)/$(libdir)/firejail/.
-	install -c -m 0755 contrib/sort.py $(DESTDIR)/$(libdir)/firejail/.
-	install -c -m 0755 contrib/syscalls.sh $(DESTDIR)/$(libdir)/firejail/.
-	install -c -m 0755 contrib/update_deb.sh $(DESTDIR)/$(libdir)/firejail/.
-	install -m 0755 -d $(DESTDIR)/$(datarootdir)/vim/vimfiles/ftdetect/.
-	install -m 0755 -d $(DESTDIR)/$(datarootdir)/vim/vimfiles/syntax/.
-	install -c -m 0644 contrib/vim/ftdetect/firejail.vim $(DESTDIR)/$(datarootdir)/vim/vimfiles/ftdetect/.
-	install -c -m 0644 contrib/vim/syntax/firejail.vim $(DESTDIR)/$(datarootdir)/vim/vimfiles/syntax/.
+	install -m 0755 -t $(DESTDIR)$(libdir)/firejail contrib/*.py contrib/*.sh
+	# vim syntax
+	install -m 0755 -d $(DESTDIR)$(datarootdir)/vim/vimfiles/ftdetect
+	install -m 0755 -d $(DESTDIR)$(datarootdir)/vim/vimfiles/syntax
+	install -m 0644 contrib/vim/ftdetect/firejail.vim $(DESTDIR)$(datarootdir)/vim/vimfiles/ftdetect
+	install -m 0644 contrib/vim/syntax/firejail.vim $(DESTDIR)$(datarootdir)/vim/vimfiles/syntax
 endif
 	# documents
-	install -m 0755 -d $(DESTDIR)/$(DOCDIR)
-	install -c -m 0644 COPYING $(DESTDIR)/$(DOCDIR)/.
-	install -c -m 0644 README $(DESTDIR)/$(DOCDIR)/.
-	install -c -m 0644 RELNOTES $(DESTDIR)/$(DOCDIR)/.
-	install -c -m 0644 etc/templates/* $(DESTDIR)/$(DOCDIR)/.
-	# etc files
+	install -m 0755 -d $(DESTDIR)$(DOCDIR)
+	install -m 0644 -t $(DESTDIR)$(DOCDIR) COPYING README RELNOTES etc/templates/*
 ifeq ($(BUSYBOX_WORKAROUND),yes)
 	./mketc.sh
 endif
-	install -m 0755 -d $(DESTDIR)/$(sysconfdir)/firejail
-	install -m 0644 -t $(DESTDIR)/$(sysconfdir)/firejail etc/profile-a-l/*.profile etc/profile-m-z/*.profile etc/inc/*.inc etc/net/*.net etc/firejail.config
-	sh -c "if [ ! -f $(DESTDIR)/$(sysconfdir)/firejail/login.users ]; then install -c -m 0644 etc/login.users $(DESTDIR)/$(sysconfdir)/firejail/.; fi;"
+	install -m 0755 -d $(DESTDIR)$(sysconfdir)/firejail
+	install -m 0644 -t $(DESTDIR)$(sysconfdir)/firejail etc/profile-a-l/*.profile etc/profile-m-z/*.profile etc/inc/*.inc etc/net/*.net etc/firejail.config
+	if [ ! -f $(DESTDIR)$(sysconfdir)/firejail/login.users ]; then \
+		install -m 0644 etc/login.users $(DESTDIR)$(sysconfdir)/firejail \
+	fi
 ifeq ($(HAVE_APPARMOR),-DHAVE_APPARMOR)
 	# install apparmor profile
-	sh -c "if [ ! -d $(DESTDIR)/$(sysconfdir)/apparmor.d ]; then install -d -m 755 $(DESTDIR)/$(sysconfdir)/apparmor.d; fi;"
-	install -c -m 0644 etc/apparmor/firejail-default $(DESTDIR)/$(sysconfdir)/apparmor.d/.
-	sh -c "if [ ! -d $(DESTDIR)/$(sysconfdir)/apparmor.d/local ]; then install -d -m 755 $(DESTDIR)/$(sysconfdir)/apparmor.d/local; fi;"
+	if [ ! -d $(DESTDIR)$(sysconfdir)/apparmor.d ]; then \
+		install -m 0755 -d $(DESTDIR)$(sysconfdir)/apparmor.d \
+	fi
+	install -m 0644 etc/apparmor/firejail-default $(DESTDIR)$(sysconfdir)/apparmor.d
+	if [ ! -d $(DESTDIR)$(sysconfdir)/apparmor.d/local ]; then \
+		install -m 0755 -d $(DESTDIR)$(sysconfdir)/apparmor.d/local \
+	fi
```

Comment:
Any reasons why we need to check this?

Reply:
> Any reasons why we need to check this?

Doesn't make much sense to me as the other `install -d` commands don't do that.

They are set to be removed on commit 5430535a2 ("build: stop checking for dir
existence on install", 2024-05-21) / PR #6366.